cromwell.examples.conf

```diff
@@ -624,6 +624,11 @@ backend {
     #      # Restrict access to VM metadata. Useful in cases when untrusted containers are running under a service
     #      # account not owned by the submitting user
     #      restrict-metadata-access = false
+    #      
+    #      # Pipelines v2 only: specify the number of times localization and delocalization operations should be attempted
+    #      # There is no logic to determine if the error was transient or not, everything is retried upon failure
+    #      # Defaults to 3
```

Comment:
I would put this into somewhere in docs too. From briefly looking, maybe "Let's get started!" on [/docs/tutorials/PipelinesApi101.md](https://github.com/broadinstitute/cromwell/blob/b830603f77eeb6af7c7721049a1d683efb3bdbbc/docs/tutorials/PipelinesApi101.md)?